compressors/cleancss.py

```diff
@@ -0,0 +1,5 @@
+from pipeline.compressors import SubProcessCompressor
+
+class CleanCSSCompressor(SubProcessCompressor):
+    def compress_css(self, css):
+        return self.execute_command('cleancss', css)
```

Comment:
Not sure a new *compressors* directory is the best place for this. Open to suggestions!